salvador/apiData.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  apiData.swift
+//  salvador
+//
+//  Created by Geury Roustand on 06.02.23.
+//
+
+import Foundation
+
+struct apiData: Codable {
```

Comment:
structs and classes should start with a capital letter (PascalCase).  Suggestion: `APIData`